src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java

```diff
@@ -284,8 +284,14 @@ public double getVoltage() {
         return Robot.getInstance().getVoltage();
     }
 
+    private boolean ignoreUpToSpeed = false;
+
     public boolean upToSpeed() {
-        return shooterSubsystem.upToSpeed();
+        return ignoreUpToSpeed ? ignoreUpToSpeed : shooterSubsystem.upToSpeed();
```

Comment:
Nit: Please either have a separate variable for the overriding value or replace the second `ignoreUpToSpeed` with `true`.